Update the Virus Scanner page (#492)
owncloud/docs-server: modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

:clamav-url: http://www.clamav.net/index.html
:icap-url: https://tools.ietf.org/html/rfc3507
:c-icap-url: https://sourceforge.net/p/c-icap/wiki/configcicap/
:kasperski-scanengine-url: https://support.kaspersky.com/ScanEngine/1.0/en-US/179682.htm
:kaspersky-icap-url: https://support.kaspersky.com/ScanEngine/1.0/en-US/179708.htm
:mcaffee-demo-url: https://www.skyhighsecurity.com/en-us/products/secure-web-gateway.html
:fortinetsandbox-url: https://www.fortinet.com/de/products/sandbox/fortisandbox

:description: When sharing files, security is a key aspect. The ownCloud Anti-Virus extension helps by protecting against malicious software like trojans or viruses. 

== Introduction

{description}

> You can get the {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension from the marketplace. It helps by protecting against malicious software like trojans or viruses. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Webinterface menu:Admin[Settings > Security].

Reviewer: ```suggestion
You can get the {oc-marketplace-url}/apps/files_antivirus[Anti-Virus] extension from the marketplace. It forwards files that are being uploaded to the ownCloud server to a  malware scanning engine before they are written to the storage. When a file is recognized as malicious, it can be logged and prevented from being uploaded to the server to ensure that files in ownCloud are free of malware. More sophisticated rules may be specified as admin in the ownCloud Web interface menu:Admin[Settings > Security].
```

Reviewer: Deleted sentence has been moved to the description.

    av_response_header --value="X-Infection-Found"
----

=== Run with Kaspersky Anti-Virus in ICAP Mode

. Install the Kaspersky ScanEngine (KAV) based on their {kasperski-scanengine-url}[instructions] and prepare KAV for running in ICAP mode.

> . Select btn:[req] from the dropdown in the Anti Virus app or use the command line or via command line:

Reviewer: ```suggestion
. Select btn:[req] from the dropdown in the Anti Virus app or use the command line:
```

// note that the original document regarding "VirusNameICAPHeader" https://support.kaspersky.com/ScanEngine/1.0/en-US/201214.htm is not available anymore (404, no cache entry) and no appropriate replacement has been found. the link above is the best possible alternative.

=== Run with McAfee Web Gateway in ICAP Mode

Follow this procedure to configure ownCloud for the McAfee Web Gateway 10.x and higher in ICAP Mode.

. Install McAfee Web Gateway based on their instructions.

. Select btn:[McAfee Web Gateway 10.x and higher] from the dropdown in the Anti Virus app.

. To use McAfee Web Gateway, set the mode to `respmod` either from the Web interface or via command line:
+
[source,bash,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set files_antivirus \
    av_request_service --value="respmod"
----

. Set the response header to `X-Virus-Name`
+
[source,bash,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set files_antivirus \
    av_response_header --value="X-Virus-Name"
----

> NOTE: McAfee provides demoversions with limited runtime for evaluation purposes. Have a look at the {mcaffee-demo-url}[McAfee Webpage] for the Web Gateway.

Reviewer: ```suggestion
NOTE: McAfee provides demo versions with limited runtime for evaluation purposes. Have a look at the {mcaffee-demo-url}[McAfee Web page] for the Web Gateway.
```

=== Run with FortiSandbox in ICAP Mode

. Install FortiSandbox based on their instructions.

> . Select btn:[Fortinet] from the dropdown in the Anti Virus app.

Reviewer: ```suggestion
. Select btn:[Fortinet] from the dropdown in the Anti-Virus app.
```

. To use FortiSandbox, set the mode to `respmod` either from the Web interface or via command line:
+
[source,bash,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set files_antivirus \
    av_request_service --value="respmod"
----

. Set the response header to `X-Virus-Name`
+
[source,bash,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set files_antivirus \
    av_response_header --value="X-Virus-Name"
----

> NOTE: Fortinet privides some demoversions of the FortiSandbox, please have a look at their {fortinetsandbox-url}[productpage] for more information.

Reviewer: ```suggestion
NOTE: Fortinet provides some demo versions of the FortiSandbox. Have a look at their {fortinetsandbox-url}[product page] for more information.
```
